lib/apple.js

```diff
@@ -27,6 +27,10 @@ module.exports.readConfig = function (configIn) {
 	}
 	config = {};
 	var configValueSet = false;
+	// Apply any default settings to Request.
+	if('requestDefaults' in configIn) {
+		request = request.defaults(configIn['requestDefaults']);
```

Comment:
My bad... I should have caught this the first time around, but would change this line to use dot-notation instead?

request = request.defaults(configIn.requestDefaults);

Sorry. I'm a little picky ;-)